src/gradualizer_cli.erl

```diff
@@ -34,6 +35,9 @@ print_usage() ->
     io:format("Usage: gradualizer [options] [PATH...]~n"),
     io:format("A type checker for Erlang/Elixir~n~n"),
     io:format("       PATH                      Files or directories to type check~n"),
+    io:format("       --                        Signals that no more options will follow. The rest~n"),
+    io:format("                                 of the arguments is treated as filenames, even if~n"),
```

Comment:
is -> are

Reply:
taken verbatim from `erl` doc, quickfixing

Reply:
...actually the "rest of sg" might be singular :)